src/elements/content-sidebar/__tests__/DocgenSidebar.test.tsx

```diff
@@ -1,17 +1,19 @@
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-// @ts-ignore
-import React, { act } from 'react';
-import { mount } from 'enzyme';
-import { MessageDescriptor, FormattedMessage } from 'react-intl';
+import React from 'react';
+import { render, screen, waitFor, fireEvent } from '@testing-library/react';
```

Comment:
use this instead https://github.com/box/box-ui-elements/blob/master/src/test-utils/testing-library.tsx 
all the RTL functions are available through that import, and it unmock react-intl this way you dont need to unmock or mock react-intl to see the text

also we dont need to wrap the component with a IntlProvider since its already wrapped via the the custom testing-library render